radiomics/imageoperations.py

```diff
@@ -13,42 +13,64 @@
 logger = logging.getLogger(__name__)
 
 
-def getHistogram(binwidth, parameterValues):
-  """
+def getBinEdges(binwidth, parameterValues):
```

Comment:
> I just now realized that numpy has its own set of strategies for calculating bins. I think it would be helpful to give user an option to access those numpy strategies. I don't think this is possible right now.

It's a good question. We discussed this in one of the hangouts some months ago (where I proposed adding the possibility for a fixed bin count). We decided then not to incorporate it to prevent confusion. Additionally, do the other options yield equally spaced bins? This is required for `numpy.digitize`. If not, we have to decide whether or not to enable this for just histogram related features (Entropy and Uniformity in firstorder)

> Related question - should we raise the question of bin calculation withe radiomics standardization effort group? It does not look like their document discusses this at the moment.

They already took it up, it's in their latest workversion (but not available online yet I think).

They already took it up, it's in their latest workversion